cassiopeia/simulator/LineageTracingDataSimulator.py

```diff
@@ -0,0 +1,25 @@
+"""
+This file stores an abstract subclass of DataSimulator, the
+LineageTracingDataSimulator. A LineageTracingDataSimulator overlays lineage
+tracing data onto a CassiopeiaTree, i.e. it sets the character states of a
+CassiopeiaTree (in particular, the character matrix).
+"""
+import abc
+from cassiopeia.data import CassiopeiaTree
+from cassiopeia.simulator.DataSimulator import DataSimulator
```

Comment:
nit: style -> add new line between `import abc` and `from cassiopeia.data import CassiopeiaTree`